doc/sphinx-guides/source/developers/version-control.rst

```diff
@@ -287,4 +287,8 @@ GitHub documents how to make changes to a fork at https://help.github.com/articl
 
 ----
 
+Develop branch and pull request 
+-------------------------------------------- 
+Please do not use your forked develop branch to push a PR and follow the `1st scenario: preparing the first pull request of Version Control Guide <https://github.com/Recherche-Data-Gouv/dataverse/blob/develop/doc/sphinx-guides/source/developers/version-control.rst>`_.
```

Comment:
I'm confused by this link. We're already in version-control.rst. Why would we link to the file we are already in? 🤔

Reply:
I redirected to the section on git commands. I just corrected the link.

Reply:
@martinAmouzou ah, thanks, I understand now and that `#` thing is a good sphinx/rst trick!

Is it ok if I move what you wrote somewhere else? And add a bit more content (for maintainers like me)?

Reply:
@pdurbin Yeah, you can do so. thanks

Reply:
Great, thanks!